fe/src/main/java/org/apache/doris/common/DuplicatedRequestException.java

```diff
@@ -20,12 +20,24 @@
 public class DuplicatedRequestException extends DdlException {
 
     private String duplicatedRequestId;
+    // save exist txn id
+    private long txnId;
 
     public DuplicatedRequestException(String duplicatedRequestId, String msg) {
```

Comment:
Should we keep this constructor? And there is possibilty that txnId can be null?

Reply:
removed